### PR TITLE
FE: Sitter Details

### LINKED ISSUE #20: FE: Profile Details page
Each Profile should have its own dedicated page that outlines their profile in detail

### PR DESCRIPTION
**Pull request by trevorjr31**
### What this PR does (required):
-Adds Sitter Details UI
-Addresses Issue 20
#20

### Screenshots / Videos (front-end only):
Screenshot (175)
 Screenshot (174)

### Any information needed to test this feature (required):
- Add required details to user profile interface:
heroImage: str
about: str
album: [str]

-Use MongoDB to add fields and populate data

### Any issues with the current functionality (optional):
N/A

### CODE REVIEW

File: client/src/components/SitterProfile/useStyles.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    marginTop: theme.spacing(0.3),
+  },
+  cardMain: {
+    height: '600px',
+    width: '475px',
+    margin: theme.spacing(16, 8),
+    padding: theme.spacing(0, 0),
+  },
+  heroImage: {
+    width: 500,
```
Comment (rajivtitus) [inline client/src/components/SitterProfile/useStyles.ts:49]: You can add an `objectFit: 'cover'` on images to preserve aspect ratios :)

**Review by rajivtitus — COMMENTED** [review]
UI looks fantastic! 💯 
Since a lot of this code was a part of the IN: Listings PR, just a minor comment for you to have a look :)

File: client/src/components/SitterProfile/useStyles.ts (hunk 1 above)
Comment (trevorjr31) [reply client/src/components/SitterProfile/useStyles.ts:49]: Change made upstream to both hero and album pics!

**Comment by trevorjr31** [comment]
👍

**Pull request merged**